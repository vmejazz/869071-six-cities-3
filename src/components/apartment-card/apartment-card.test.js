import React from "react";
import renderer from "react-test-renderer";
// import ApartmentCard from "./apartment-card";

// const placeOffer = {
//   id: 1,
//   title: `New Perfect Apartment`,
//   price: 500,
//   srcImg: `./img/apartment-03.jpg`
// };

describe(`Test ApartmentCard component`, () => {

  it(`<ApartmentCard /> should render apartment card `, () => {
    const tree = renderer
      .create(
          <h1></h1>
          // <ApartmentCard
          //   placeOffer={placeOffer}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

});
