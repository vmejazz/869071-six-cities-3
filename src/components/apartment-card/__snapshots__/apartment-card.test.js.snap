// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ApartmentCard component <ApartmentCard /> should render apartment card  1`] = `<h1 />`;
